docs/testing.md

```diff
@@ -47,15 +49,17 @@ Sauce Labs can be used to run tests remotely in the cloud.
 
 7. Run the tests with `gulp test:acceptance`.
 
-    > Note: If you want to temporarily disable testing on Sauce Labs,
-    run the command as: `gulp test:acceptance --sauce=false`.
+    !!! Note: 
```

Comment:
TRIVIAL: In the past we used `>` for notes to separate and highlight them from the main body content, e.g.:

```
> NOTE:
   Blah, blah, blah
```